packages/material-ui-lab/src/themeAugmentation/props.d.ts

```diff
@@ -34,6 +42,25 @@ export interface LabComponentsPropsList {
   MuiToggleButtonGroup: ToggleButtonGroupProps;
   MuiTreeItem: TreeItemProps;
   MuiTreeView: TreeViewProps;
+  MuiPickersDay: PickersDayProps;
+  MuiDayPicker: DayPickerProps;
+  MuiMonthPicker: MonthPickerProps;
+  MuiPickersCalendarSkeleton: CalendarSkeletonProps;
```

Comment:
?
```suggestion
  MuiCalendarSkeleton: CalendarSkeletonProps;
```

Reply:
The other way around sounds fine too

```suggestion
MuiPickersCalendarSkeleton: PickersCalendarSkeletonProps;
```